apps/src/templates/PuzzleRatingButtons.jsx

```diff
@@ -57,6 +69,10 @@ class PuzzleRatingButtons extends Component {
     };
   }
 
+  static propTypes = {
+    useLegacyStyles: React.PropTypes.bool,
```

Comment:
The screenshots in the description show the old style, the new style is only used in the gamification dialogs. I prefer the existing style with these dialogs, but here's how the new buttons would look:
![screenshot from 2017-07-26 12-01-38](https://user-images.githubusercontent.com/1070243/28638605-3411ad94-71fa-11e7-9da8-3f40e796d0b9.png)